skins/oasis/modules/BodyController.class.php

```diff
@@ -456,13 +456,25 @@ public function executeIndex() {
 		$this->setVal( 'displayHeader', !$this->wg->HideNavigationHeaders );
 
 		$title = $this->wg->Title->getPrefixedDBkey();
-		$enableArticleVideo =
-			$this->wg->enableArticleVideo && isset( $this->wg->articleVideoFeaturedVideos[$title] ) &&
+		$enableArticleFeaturedVideo =
+			$this->wg->enableArticleFeaturedVideo && isset( $this->wg->articleVideoFeaturedVideos[$title] ) &&
 			$this->isFeaturedVideosValid( $this->wg->articleVideoFeaturedVideos[$title] );
-		$this->setVal( 'enableArticleVideo', $enableArticleVideo );
+		$this->setVal( 'enableArticleFeaturedVideo', $enableArticleFeaturedVideo );
```

Comment:
Why do we need to set those variables in `BodyController`? Is it possible to make those checks inside `ArticleVideo` extension?